lib/app/data/source/random_number_data_source.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:get/get.dart';
+
+class RandomNumberDataSource extends GetConnect implements GetxService {
+  Future<int> get randomNumber async {
+    final response = await get(
+      'http://www.randomnumberapi.com/api/v1.0/randomredditnumber?min=0&max=100&count=1',
```

Comment:
Should be `get('/randomredditnumber')`. Url should be defined in the baseUrl field, in the constructor. The query parameters should be passed to query parameter.
```
{
'min': 0,
'max': 100,
'count': 1,
}
```